bot_file.py

```diff
@@ -404,6 +406,34 @@ def elm_progress_on(self, channel: str, branch_name: str) -> None:
 
         self.send_channel_message(channel, message)
 
+    def find_elm_017_matches(self, channel: str, filename_pattern: str) -> None:
+        """ give a filename of elm to get me to tell you how it looks on master """
+
+        self.repo.get_ready()
+        message = "We have found the following filenames:\n"
+
+        filenames = self.repo.get_files_for_017(filename_pattern)
```

Comment:
:art: if you look only at this line, not evident that we're fetching elm stuff:

- `self.repo.get_elm_files_for_017`?
- or `self.elm_repo.get_files_for_017`?

and/or make ElmRepo not inherit from Repo? (can wait until we have non-elm stuff)

```python
class RepoBase(object): ...
class ElmRepoMixin(object): ...

class OurRepo(ElmRepoMixin, RepoBase): ...
```